src/Contact.scss

```diff
@@ -2,9 +2,30 @@
     text-align: left;
     padding-inline-start: 0px;
     background-color: #FFF;
+    flex-shrink: 0;
     li {
         list-style-type: none;
         margin-top: 10px;
     }
+
+    @media (max-width: 900px) {
+        width: 377px;
+        height: 202px;
+        display: flex;
+        flex-direction: column; 
+        align-items: flex-start;
+        
+    }
+
+    @media (min-width: 901px) {
+        display: flex;
+        flex-direction: row; 
+        justify-content: space-around;
+        align-items: center;
+        display: flex;
+        padding: 31px 397px;
+        gap: 10px;
+        align-self: stretch;
+    }
     
```

Comment:
you do not need two different media one for mobile and one for desktop.
the regular style needs to be to mobile and the media need to be for the desktop style changes

for example: 
```
.contact-us {

     display: flex;
      flex-direction: row; 
      justify-content: space-around;
      align-items: center;

// this will be the changes for the desktop 
    @media (max-width: 900px) {
        flex-direction: column; 
        align-items: flex-start;
    }

}
```

Reply:
fixed